seed_tl.ps1

```diff
@@ -1,17 +1,51 @@
-﻿$localVersion = 2.5
-$configReq = 2
+﻿$localVersion = 3.0
+$configReq = 3
 
 $latestdl = 'https://od.lk/fl/NjJfMzM2NDI2M18'
 $elevate = "false"
 
 #######################################################################################################################################
 # Self-elevate the script if required
 if (-Not ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole] 'Administrator')) {
- if ([int](Get-CimInstance -Class Win32_OperatingSystem | Select-Object -ExpandProperty BuildNumber) -ge 6000 -and $elevate -eq "true") {
-  $CommandLine = "-File `"" + $MyInvocation.MyCommand.Path + "`" " + $MyInvocation.UnboundArguments
-  Start-Process -FilePath PowerShell.exe -Verb Runas -ArgumentList $CommandLine
-  Exit
- }
+    if ([int](Get-CimInstance -Class Win32_OperatingSystem | Select-Object -ExpandProperty BuildNumber) -ge 6000 -and $elevate -eq "true") {
+        $CommandLine = "-File `"" + $MyInvocation.MyCommand.Path + "`" " + $MyInvocation.UnboundArguments
+        Start-Process -FilePath PowerShell.exe -Verb Runas -ArgumentList $CommandLine
+        Exit
+    }
+}
+
+function MoveResize-Console-Window
+{
```

Comment:
New resize and move code.

Reply:
might have to redo this in the future.. it currently resizes the CURRENT Window.. so if you start the script and then switch to a different window, it resizes THAT window, not the powershell one.. will research fixes